audio/portaudio/Portfile

```diff
@@ -3,8 +3,10 @@
 PortSystem          1.0
 
 name                portaudio
-version             19.20140130
-revision            2
+set major           19
+set minor           6
+set reldate         20161030
+version             ${major}.${reldate}
 categories          audio devel
```

Comment:
Ah, okay, it looks like this will work now. I don't love it but whatever, it is consistent so the system will not break because of a change in the numbering scheme.